pkgs/top-level/all-packages.nix

```diff
@@ -26357,10 +26357,14 @@ in
 
   protonmail-bridge = callPackage ../applications/networking/protonmail-bridge { };
 
-  protonvpn-cli = callPackage ../applications/networking/protonvpn-cli { };
+  protonvpn-community-cli = callPackage ../applications/networking/protonvpn-community-cli { };
+
+  protonvpn-legacy-gui = callPackage ../applications/networking/protonvpn-legacy-gui { };
```

Comment:
I think we can safely drop this.

Reply:
Thanks for the review. To be sure what you suggest is to completely remove the "old packages"?

Reply:
I thought the new packages replace the old one. Is there a good reason to continue to use the old package if it is no longer being developed?

Reply:
I believe that they renamed it the repository to indicate that the old package would not be officially supported (I'm not sure to what extent is was officially supported previously). They've also recently included a notice in the old package, informing users on systems with networkmanager, about the new package. The only reason I'm slightly skeptical about removing the old packages is that I don't know the extent of the usage of these packages in the community. We could possibly give the maintainers of the old packages a some time to share their insights.

Reply:
Hmmm.. I am not sure what is the right thing todo here. People who do not use networkmanager can't use the new package?

Reply:
No they would be unable to. We could use the old names and add `official` at the end as such:
- `protonvpn-cli-official`
- `protonvpn-gui-official`

Reply:
I think that would be the best option we currently have.

Reply:
I've added official to the new packages, and reverted to name changes of the old packages.